spec/lib/audit/catalog_to_moab_class_spec.rb

```diff
@@ -0,0 +1,73 @@
+require_relative '../../../lib/audit/catalog_to_moab.rb'
+require_relative '../../load_fixtures_helper.rb'
+
+RSpec.describe CatalogToMoab do
+  let(:last_checked_version_b4_date) { (Time.now.utc - 1.day).iso8601 }
+  let(:storage_dir) { 'spec/fixtures/storage_root01/moab_storage_trunk' }
+
+  context '.check_version_on_dir' do
+    include_context 'fixture moabs in db'
+    let(:subject) { described_class.check_version_on_dir(last_checked_version_b4_date, storage_dir) }
+
+    it 'creates an instance and calls #check_catalog_version' do
+      c2m_mock = instance_double(described_class)
+      allow(described_class).to receive(:new).and_return(c2m_mock)
+      expect(c2m_mock).to receive(:check_catalog_version).exactly(6).times # FIXME: will reduce to 3 for PR #497
```

Comment:
note that I added FIXME to this line and 21 and 26 below  because ... it should only be 3 times, yes???